src/main/java/com/app/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,29 @@
 package com.app.onlinebookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.validator.constraints.Length;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
+    @NotBlank

```

Comment:
You can change these annotations using just one annotation 
[see here ](https://www.baeldung.com/java-bean-validation-not-null-empty-blank)